### PR TITLE
Issue 159: Update review pre-visit

### PR DESCRIPTION
**Pull request by rmroot**
connects #159 

review pre-visit page updated to match changes to data.

### CODE REVIEW

File: src/app/setup-wizard/pre-visit/review-pre-visit-setup/company-details-summary/company-details-summary.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@ <h5 class="bold w-100 text-center">
     <fa-icon [icon]="faBuilding" class="me-2"></fa-icon> {{company.generalInformation.name}}
 </h5>
 
-<table class="table table-bordered table-hover">
+<table class="table table-bordered table-hover table-sm">
     <tbody>
         <tr>
             <td class="w-25">
```
Comment (RLiNREL) [inline src/app/setup-wizard/pre-visit/review-pre-visit-setup/company-details-summary/company-details-summary.component.html:8]: It looks the address is not added in. better to comment it out if not included in alpha release,

File: src/app/setup-wizard/pre-visit/review-pre-visit-setup/facility-details-summary/facility-details-summary.component.html [hunk 2]
```diff
@@ -2,7 +2,7 @@ <h5 class="bold w-100 text-center">
     <fa-icon [icon]="faIndustry" class="me-2"></fa-icon> {{facility.generalInformation.name}}
 </h5>
 
-<table class="table table-bordered table-hover">
+<table class="table table-bordered table-hover table-sm">
     <tbody>
         <tr>
             <td class="w-25">
```
Comment (RLiNREL) [inline src/app/setup-wizard/pre-visit/review-pre-visit-setup/facility-details-summary/facility-details-summary.component.html:8]: same with company address, suggest to comment it out if not included in the alpha release

File: src/app/setup-wizard/pre-visit/review-pre-visit-setup/company-details-summary/company-details-summary.component.html (hunk 1 above)
Comment (rmroot) [reply src/app/setup-wizard/pre-visit/review-pre-visit-setup/company-details-summary/company-details-summary.component.html:8]: fixed

File: src/app/setup-wizard/pre-visit/review-pre-visit-setup/facility-details-summary/facility-details-summary.component.html (hunk 2 above)
Comment (rmroot) [reply src/app/setup-wizard/pre-visit/review-pre-visit-setup/facility-details-summary/facility-details-summary.component.html:8]: fixed

**Review by RLiNREL — APPROVED** [review]

**Pull request merged**